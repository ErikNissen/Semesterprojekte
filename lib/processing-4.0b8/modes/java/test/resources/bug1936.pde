version https://git-lfs.github.com/spec/v1
oid sha256:e6304cc1f02f4a723711cb14872ea8254c973c5fba0fb8dccd58f0841460b868
size 31
