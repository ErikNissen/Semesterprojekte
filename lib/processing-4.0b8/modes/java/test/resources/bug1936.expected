version https://git-lfs.github.com/spec/v1
oid sha256:464e97b8518ec731cd3f76d74b68e40f5e1ab6e0c011f818bf41b13f67966de4
size 764
